src/CSharp/EasyMicroservices.Serialization.MemoryPack/EasyMicroservices.Serialization.MemoryPack.csproj

```diff
@@ -0,0 +1,15 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
```

Comment:
does it not support another dotnet version? if it supports it, then please add multiple target frameworks.

Reply:
Other versions are also supported, but according to the document of this package, it is better to use this package in version 7.NET